relay-log/src/setup.rs

```diff
@@ -287,12 +298,17 @@ pub fn init(config: &LogConfig, sentry: &SentryConfig) {
         .init();
 
     if let Some(dsn) = sentry.enabled_dsn() {
+        let server_name = env::var(SERVER_NAME_VAR)
+            .ok()
+            .map(Cow::from)
+            .or_else(|| sentry.server_name.clone());
```

Comment:
This prefers the environment variable if set, not sure if the other way around makes more sense.

Reply:
We have [`apply_override`](https://github.com/getsentry/relay/blob/master/relay-config/src/config.rs#L1631) at the `Config` for env overrides, probably a better place to put it.

Reply:
Oh nice